libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -22,6 +22,13 @@ const AP_Param::GroupInfo AP_Mission::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("RESTART",  1, AP_Mission, _restart, AP_MISSION_RESTART_DEFAULT),
 
+    // @Param: OPTIONS
+    // @DisplayName: Mission options bitmask
+    // @Description: Bitmap of what options to use. This values is made up of the sum of each of the mission options you require.  The individual bits are CLEAR_MISSION=1
+    // @Values: 0:Clear Mission, 1:Retain Mission
```

Comment:
This should be a @<span></span>Bitmask, not Values. Also, 0 should be current behavior (retain mission) and 1 clear mission.